charts/kube-prometheus-stack/values.yaml

```diff
@@ -413,6 +413,26 @@ alertmanager:
     interval: ""
     selfMonitor: true
 
+    ## SampleLimit defines per-scrape limit on number of scraped samples that will be accepted.
```

Comment:
Would be nice to have it like this:

```
  scrapeLimits: {}
      # sampleLimit: 0
      # targetLimit: 0
      # labelLimit: 0
      # labelNameLengthLimit: 0
      # labelValueLengthLimit: 0
```

Would safe space and do not add this by default, keeping current behavouir.

Reply:
I've wanted to follow same behavior like in https://github.com/prometheus-community/helm-charts/pull/2609 and https://github.com/prometheus-community/helm-charts/pull/2610